### DISCUSSION TITLE
Vote for Wrappers

### DISCUSSION THREAD

**Discussion by 0xflotus** [supabase/supabase#11479] [Feature Requests]
On 15th December 2022, [supabase released Wrappers](https://supabase.com/blog/postgres-foreign-data-wrappers-rust), a framework to build FDWs in Rust. 

Maybe we can here collect some ideas for Wrappers in the future. 

First one from me is [Redis](https://redis.com/) or [KeyDB](https://docs.keydb.dev/)

**Comment by burmecia** [comment]
My vote is for [MySQL](https://www.mysql.com/)

**Comment by burmecia** [comment]
another vote is for [OpenAI GPT](https://openai.com/)

**Comment by RihanArfan** [comment]
GraphQL and OData (RESTful) would let me use Supabase to build an application for our enterprise which extends Microsoft's Graph API (RESTful via OData) and our headless CMS (GraphQL)

Similar alternatives where it's possible:
- Harusa https://hasura.io/docs/latest/remote-schemas/remote-relationships/index/
- WunderGraph https://docs.wundergraph.com/docs/supported-data-sources

**Comment by governoze** [comment]
RestAPI or GraphQL API such as available on https://hasura.io/docs/latest/data-federation/overview/

**Comment by riderx** [comment]
tinybird To ingest data with a scaling solution

**Comment by sanchezcarlosjr** [comment]
Oracle.
